### PR TITLE
[13] Style the Product Details Page using CSS

### PR DESCRIPTION
**Pull request by marindotib**
product page web view
product page tablet view
product page mobile view

### CODE REVIEW

**Comment by adaugochi** [comment]
well done. According to the mockup the image and this 
image isn't far apart

**Comment by marindotib** [comment]
I already modified the PR
web view of product page
mobile view  for product page
tablet view of product page

File: submissions/boladeMarindoti/css/styles2.css [hunk 1]
```diff
@@ -0,0 +1,240 @@
+body{
+    font-family: Arial, Helvetica, sans-serif;
```
Comment (adaugochi) [inline submissions/boladeMarindoti/css/styles2.css:2]: please rename this file to `product.css` instead

Comment (marindotib) [reply submissions/boladeMarindoti/css/styles2.css]: I have  renamed the file to product.css

**Pull request merged**